### ISSUE TITLE
DToC CSS needs to render lg and l tags for poetry

### ISSUE THREAD

**Issue by SusanBrown** [sgsinclair/Voyant#350]
We need line breaks at the end of the l (letter ell) tags
and a space before and after lg tags

**Comment by ajmacdonald** [comment]
@SusanBrown can you please clarify what you mean by "a space before and after lg tags" ? Are you talking about extra margin space or a whitespace character?

**Comment by SusanBrown** [comment]
I was just checking to see that Regenerations doesn’t have any lines of poetry, but it doesn’t seem to.

Sorry, should have said it more clearly. lg should be set off like a block quotation (same margin indent) and also have a blank line above and below.

**Comment by ajmacdonald** [comment]
I'm not sure what you meant, but Regenerations does have some poetry in it.

The lg tags (except for 2 of them in my copy) are already inside of block quotations, so do you want an additional margin indent for lg?

**Comment by SusanBrown** [comment]
No, maybe the problem is that they should have been inside a block quote to begin with. Will experiment further.

**Comment by sgsinclair** [comment]
A further question is if vanilla voyant should add html breaks for these tags (when not used in dtoc), my inclination would be to say yes.

**Comment by SusanBrown** [comment]
Me too.
